pipelines/perf-eval/Controller/job-scheduling.yml

```
@@ -0,0 +1,66 @@
+trigger: none
+schedules:
+  - cron: "30 1 */2 * *" # Every 2 days at 1:30 AM
+    displayName: "1:30 AM every 2 days"
+    branches:
+      include:
+        - main
+        - vitto/kwok-cl2 # to be removed after the PR is merged
+    always: true
+variables:
+  SCENARIO_TYPE: perf-eval
+  SCENARIO_NAME: job-scheduling
+  SCENARIO_VERSION: main
+stages:
+  - stage: azure_eastus2
+    dependsOn: []
+    jobs:
+      - template: /jobs/competitive-test.yml
+        parameters:
+          cloud: azure
+          regions:
+            - eastus2
+          engine: clusterloader2
+          engine_input:
+            image: "ghcr.io/azure/clusterloader2:v20241022"
```

We have recently merged upstream changes into perf-tests repo and generated a new image for cl2. Can you use the latest image and run your tests on the latest image?
```suggestion
            image: "ghcr.io/azure/clusterloader2:v20250423"
```